roboto/types.py

```diff
@@ -58,23 +58,55 @@ class ChatPhoto:
     """Information for fetching the chat picture."""
 
     small_file_id: str
+    small_file_unique_id: str  #don't know if changed or made by design
     big_file_id: str
+    big_file_unique_id: str  #don't know if changed or made by design
+
+
+# should this be private? (_ChatType)
+class ChatType(Enum):
```

Comment:
This is an interesting change, but `from_json` does not support enumerations yet. There is good reason: if a value is added to the possible inputs, a change to the Telegram API breaks older versions of the library, breaking running bots.

Notice that this is not true for values that are sent __to__ the API: if the user is constrained to choosing among a set of values and the API starts supporting a new one, the user's code runs fine as it always had. He just needs to upgrade `roboto` if he wants to use a new one.

If the value is received __from__ the API, though, we will break, since `from_json` will try to instantiate an `Enum` type with a value that is not supported. Therefore, if we ever wish to encode this, we will have to use something such as `Union[ChatType, str]` and return the raw value if we aren't aware of it. This may prove to be too much of a hassle, though.